.travis.yml

```diff
@@ -28,6 +28,10 @@ jobs:
     dist: bionic
     env: S2N_LIBCRYPTO=openssl-1.1.1 BUILD_S2N=true TESTS=integration GCC_VERSION=6
 
+  - os: linux
+    dist: trusty
+    env: S2N_LIBCRYPTO=openssl-1.1.1 BUILD_S2N=true GCC_VERSION=4
```

Comment:
The package name for these older gcc have the minor versions in the package name, might need to update this to `GCC_VERSION=4.8`  (see failure [here](https://travis-ci.org/awslabs/s2n/jobs/633510426?utm_medium=notification&utm_source=github_status))